### ISSUE TITLE
`asdf install bitwarden lasts` fails

### ISSUE THREAD

**Issue by laidbackware** [vixus0/asdf-bitwarden#3]
When using the latest version of asdf, the install fails if latest is passed instead of a version. I've had a dig through the code and TBH I can't see what the root cause is and asdf makes if difficult to debug as it doesn't handle returned errors properly. I think it has something to do with the way that the list-all returns, maybe there is a string formatting issue.

```sh
$ asdf install bitwarden latest
No compatible versions available (bitwarden [0-9])
bitwarden  is already installed
```

**Comment by p-hash** [comment]
Similar error from [rtx](https://github.com/jdxcode/rtx) 

```
$ RTX_DEBUG=1 rtx install bitwarden latest
[DEBUG] rtx::config: Config:
  Files: ~/.tool-versions
  Installed Plugins: awscli, bitwarden, direnv, nodejs, python
[DEBUG] rtx::cli: rtx install bitwarden latest
[DEBUG] rtx::toolset::builder: Toolset: bitwarden@latest, direnv@latest, python@latest, awscli@latest, nodejs@latest
[DEBUG] rtx::runtimes: install bitwarden@latest latest
rtx bitwarden@latest installing
[DEBUG] rtx::cmd: $ /home/paul/.local/share/rtx/plugins/bitwarden/bin/install 
rtx bitwarden@latest Downloading bw version latest from https://github.com/bitwarden/cli/releases/download/latest/bw-linux-latest.zip
rtx bitwarden@latest Error: bw version latest not found
rtx bitwarden@latest error ✗
Error: 
   0: [bitwarden] script exited with non-zero status: exit code 1

Location:
   src/plugins/script_manager.rs:182

Backtrace omitted. Run with RUST_BACKTRACE=1 environment variable to display it.
Run with RUST_BACKTRACE=full to include source snippets.
```

**Comment by michaeldelago** [comment]
The [bitwarden/cli](https://github.com/bitwarden/cli) repo has been archived as of May of last year

> Archived
> This repository is archived, please go to https://github.com/bitwarden/clients for future development.



This plugin should be updated to use the [new repo](https://github.com/bitwarden/clients)

**Comment by vixus0** [comment]
Resolved by #4 hopefully.
